app/istqb-ct-game-complete-guide/istqb-ct-game-complete-guide.css

```
@@ -0,0 +1,975 @@
+            body::after {
+                content: '';
+                position: fixed;
+                inset: 0;
+                background-image:
+                    linear-gradient(rgba(0, 212, 255, 0.03) 1px, transparent 1px),
+                    linear-gradient(90deg, rgba(0, 212, 255, 0.03) 1px, transparent 1px);
+                background-size: 40px 40px;
+                pointer-events: none;
+                z-index: 9998;
+            }
+            @keyframes fadeInDown {
```

Stylelint (17.12.0)

[error] Expected keyframe name "fadeInDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)